src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -1342,4 +1344,55 @@ namespace TerminalAppLocalTests
         });
     }
 
+    void TabTests::TestClampSwitchToTab()
+    {
+        Log::Comment(L"Preview a color scheme. Make sure it's applied, then committed accordingly");
+

```

Comment:
```suggestion
```

well not exactly, but we could also just remove the comment entirely

after all, taef logs the test name :smile:

Reply:
```suggestion
        Log::Comment(L"Test that switching to a tab index higher than the number of tabs just clamps to the last tab.");

```